src/main/scala/com/mwz/sonar/scala/ScalaPlugin.scala

```diff
@@ -47,6 +47,13 @@ object Scala {
   private val DefaultScalaVersion = ScalaVersions.Scala_2_11.toString()
   private val SourcesPropertyKey = "sonar.sources"
   private val DefaultSourcesFolder = "src/main/scala"
+  private val Test: Option[String] = Some("test")
+
+  def test() = {
+    Test.get
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use of Option.get [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.sonar-scala.com/coding_rules#rule_key=scapegoat%3Acom.sksamuel.scapegoat.inspections.option.OptionGet)